# This message is for detection results in a generic format
Header header
rasberry_perception/BoundingBox[] bounding_boxes
rasberry_perception/SegmentationLabel[] instances
string[] class_labels